MergeHRISClient/api/teams_api.py

```diff
@@ -171,8 +164,6 @@ def __teams_list(
                         (datetime,),
                     'cursor':
                         (str,),
-                    'expand':
```

Comment:
did you generate this sdk with the no-expands urls? I dont think we want to remove all of these different expand params